src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -23,17 +23,17 @@ class CustomizedAxisTick extends React.Component {
 
 function getDates(startDate, endDate, days) {
   const dates = [];
-  let currentDate = new Date(startDate);
-  const endingDate = new Date(endDate);
+  let currentDate = moment(startDate).replace(/-/g, '/');
+  const endingDate = moment(endDate).replace(/-/g, '/');
   const addDaysToDate = (date) => {
-    const newDate = new Date(date.valueOf());
-    newDate.setDate(newDate.getUTCDate() + days);
+    const newDate = moment(date).valueOf();
+    newDate.set(newDate.date() + days);
     return newDate;
   };
   while (currentDate <= endingDate) {
```

Comment:
i suspect if this will work if `currentDate` and `endingDate` are both `moment` objects, if not, this may ends up in an infinite loop

maybe shuold consider https://momentjs.com/docs/#/query/is-same-or-before/